packages/frontend/app/styles/components/api-version-check.scss

```diff
@@ -1,10 +1,6 @@
 @use "../ilios-common/colors" as c;
 @use "../ilios-common/mixins" as m;
 
-.api-version-check-warning {
```

Comment:
just double-checking on this style class removal - is this intentional?

Reply:
Yes. Forgot to put that in the commit. We're not using it anywhere.